svthalia/concrexit
# Add country field to partner

Closes #1348 


### Summary
Made partner address values the same as in users.

### How to test
Steps to test the changes you made:
1. Go to partner admin
2. Set the country to a foreign country
3. Go to the partner page
4. See the partner's country and it's location on the map
